app/services/menu/loader.rb

```diff
@@ -13,11 +13,11 @@ def self.load
 
       Manager.map :user_menu do |menu|
         menu.item :my_account,
-                  :caption => N_('My account'),
+                  :caption => N_('My Account'),
```

Comment:
The change should be done in the string so that translators see the correct string to translate, and capitalize accordingly (at least in languages that have capitalization). `titelize` capitalizes all words, including those that don't need capitalization according to the style guide.